src/Components/Server/test/Circuits/CircuitIdFactoryTest.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.AspNetCore.Components.Server.Circuits;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.WebUtilities;
+using Microsoft.Extensions.Options;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Components.Server.Tests.Circuits
```

Comment:
remove `.Tests`

It might be a good idea to just set the base namespace of the project so we don't keep fighting it.